root/static/scripts/common/components/Autocomplete2.js

```diff
@@ -47,15 +47,13 @@ import type {
 const INITIAL_STATE: State = {
   highlightedIndex: 0,
   indexedSearch: true,
-  inputTimeout: null,
   inputValue: '',
   isOpen: false,
   items: EMPTY_ARRAY,
   page: 1,
   pendingSearch: null,
   selectedItem: null,
   statusMessage: '',
-  xhr: null,
```

Comment:
This (and `inputTimeout`) aren't stored in state at all, they're stored on the instance. Flow just didn't notice until exact object types were enabled.